x-pack/plugins/apm/server/lib/service_map/get_service_anomalies.ts

```diff
@@ -26,7 +26,7 @@ async function getApmAnomalyDetectionJobs(
     const { jobs } = await ml.anomalyDetectors.jobs(APM_ML_JOB_GROUP_NAME);
     return jobs;
   } catch (error) {
-    if (error.statusCode === 404) {
+    if (error.statusCode === 404 || error.statusCode === 403) {
       return [];
     }
     throw error;
```

Comment:
Question is: should we ever throw aka crash service maps because of an issue with ml integration?

Reply:
There might be other things that we haven't considered that could bring service maps down.

Reply:
At least the one that calls `getApmAnomalyDetectionJobs` should catch any errors

Reply:
We should do the same with alerting: catching errors and log them but not bringing the whole of APM down if they fail for some reason.